src/MoBi.UI/Views/EditIndividualAndExpressionConfigurationsView.cs

```diff
@@ -130,18 +161,23 @@ public void BindTo(IndividualSelectionDTO individualSelectionDTO)
       public void AddUnusedExpression(ITreeNode treeNodeToAdd)
       {
          projectExpressionsTree.TreeView.AddNode(treeNodeToAdd);
-         projectExpressionsTree.TreeView.SelectNode(treeNodeToAdd);
+         projectExpressionsTree.TreeView.Selection.Add(treeListNodeFor(projectExpressionsTree.TreeView, treeNodeToAdd));
       }
 
       public void AddUsedExpression(ITreeNode treeNodeToAdd)
       {
          simulationExpressionsTree.AddNode(treeNodeToAdd);
-         simulationExpressionsTree.SelectNode(treeNodeToAdd);
+         simulationExpressionsTree.Selection.Add(treeListNodeFor(simulationExpressionsTree, treeNodeToAdd));
+      }
+
+      private TreeListNode treeListNodeFor(UxTreeView treeView, ITreeNode treeNodeToAdd)
+      {
+         return treeView.Nodes.Single(x => Equals(x.Tag, treeNodeToAdd));
```

Comment:
Check methods in UxTreeView. Isn't there anything? 
Also why SIGNLE? It needs to traverse the whole tree to verify unicity